prime-dotnet-webapi/Models/AccessClauses/SantaClaus.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    [NotMapped]
+    public class SantaClaus : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public string WhatDoesSantaSay
+        {
+            get => "Ho ho ho ho!!! Merrrry Christmas!";
+        }
+
+        public string WhatDoesSantaWant
+        {
+            get => "Milk and Cookies!";
+        }
+
+        public ICollection<Enrollee> NaughtyList {get; set;}
+        public ICollection<Enrollee> NiceList {get; set;}
```

Comment:
Default enrollee candidates for the NiceList: @AnaisHebert, @james-hollinger

Reply:
You forgot @mtpultz